ietf/templates/includes/footer-iab.html

```diff
@@ -0,0 +1,35 @@
+<div 
+    style="
+        background-image: url('static/img/background-dark.jpg');
+        height: 48px;
+    "
+></div>
+<footer class="bg-navy-iab text-light py-3">
+    <div class="container">
+        <ul class="nav d-flex">
+            {% for item in settings.utils.FooterLinks.footer_link_items.all %}
+                <li class="navbar-text-iab footer-link-iab me-2"><a class="nav-link text-light" href="{{ item.link }}">{{ item.title }}</a></li>
+            {% endfor %}
+            <li class="navbar-text-iab footer-link-iab ms-auto p-2">
+                <a href="https://twitter.com/intarchboard" rel="me">
+                    <img src="/static/img/twitter-icon.svg"></img>
+                </a>
+            </li>
+            <li class="navbar-text-iab footer-link-iab p-2">
+                <a href="https://www.linkedin.com/company/internet-architecture-board/" rel="me">
+                    <img src="/static/img/linkedin-icon.svg"></img>
+                </a>
+            </li>
+            <li class="navbar-text-iab footer-link-iab p-2">
+                <a href="https://youtube.com/playlist?list=PLC86T-6ZTP5jeMlmiwMRcJ68N5IlWpxuQ" rel="me">
+                    <img src="/static/img/youtube-icon.svg"></img>
+                </a>
+            </li>
+            <li class="navbar-text-iab footer-link-iab p-2">
+                <a href="https://mastodon.online/@intarchboard" rel="me">
+                    <img src="/static/img/mastodon-icon.svg"></img>
+                </a>
```

Comment:
TIL `rel="me"`.

Reply:
These links have no accessible text. And the images have no alt attribute. 

You should probably:
- add `aria=label="Twitter"` (etc) to each link, and
- add empty alt attribute (`alt=""`) to each img

Reply:
Also, image tags are usually self-closing (`<img />`)though I don't know how much that matters 🤷

Reply:
all those changes are made